style.css

```
@@ -0,0 +1,12 @@
+@charset "UTF-8";
+#square {
+    width: 256px;
+    height: 256px;
+    background: #FF0000;
+    }
+
+#blue {
+    width: 256px;
+    height: 256px;
+    background: #004EFF;
+    }   
```

余談ですが
```suggestion
#square, #blue {
    width: 256px;
    height: 256px;
 }
 
 #square{
   background: #FF0000;
 }

#blue {
    background: #004EFF;
}   
```
とまとめて書くことができます＆推奨しています